pkg/timeline/adaptor/repository/dummy.ts

```diff
@@ -23,15 +23,20 @@ export class InMemoryTimelineRepository implements TimelineRepository {
       (note) => note[1].getAuthorID() === accountId,
     );
 
+    // NOTE: filter DIRECT notes
```

Comment:
```suggestion
    // NOTE: filter out DIRECT notes
```
filter だとそれだけを通すという逆の意味になるので, 取り除くなら filter out かと